### PR TITLE
Add quickstart guides for new users and CI tests

### PR DESCRIPTION
**Pull request by michaelsevilla**
Added `dmake` description and all tutorials to the wiki.

Addresses #21.

### CODE REVIEW

File: builder-base/scripts/generate_daemon_image [hunk 1]
```diff
@@ -58,7 +58,7 @@ fi
 # means that we'll skip committing the container
 
 # get the delta
-delta=`rsync -ani --size-only --omit-dir-times --exclude=base --out-format='%n' $CEPH_INSTALL_PATH/ $CEPH_INSTALL_PATH/base`
```
Comment (michaelsevilla) [inline builder-base/scripts/generate_daemon_image]: When the image is already built and we want to re-compile and build a new image, this command returns:

```
delta=
```

But removing `--size-only` we re-copy all the binaries but I feel like this is a small price to pay... Unless it creates a  layering problem where diffs of our Docker images in our registry can't be pulled easily anymore.

File: builder-base/scripts/generate_daemon_image [hunk 2]
```diff
@@ -58,7 +58,7 @@ fi
...
+delta=`rsync -ani --omit-dir-times --exclude=base --out-format='%n' $CEPH_INSTALL_PATH/ $CEPH_INSTALL_PATH/base`
```
Comment (michaelsevilla) [inline builder-base/scripts/generate_daemon_image:61]: When the image is already built and we want to re-compile and build a new image, this command returns:

```
delta=
```

By removing `--size-only` we re-copy all the binaries but I feel like this is a small price to pay... Unless it creates a  layering problem where diffs of our Docker images in our registry can't be pulled easily anymore.

**Comment by michaelsevilla** [comment]
Commit 6630717 addresses #11.

The build results for the Travis CI tests are linked to my `docker-cephdev` fork and can viewed [here](https://travis-ci.org/michaelsevilla/docker-cephdev/builds/134306138). Eventually, we need to either transfer them to your repo or create a new one under the [System's Research Lab's repo](https://github.com/systemslab).

**Comment by ivotron** [comment]
this is awesome!

**Pull request merged**